README.md

```diff
@@ -1,26 +1,26 @@
-# Stonesoup documentation
+# App Studio documentation
 
-Stonesoup is an effort to automate build and delivery of Red Hat products. With Stonesoup, you have access to single, simple workflow for developing, testing, and releasing containerized applications while ensuring compliance with enterprise security standards.
+App Studio is an effort to automate build and delivery of Red Hat products. With App Studio, you have access to single, simple workflow for developing, testing, and releasing containerized applications while ensuring compliance with enterprise security standards.
```

Comment:
I believe we can use {ProductName} attribute instead of App Studio.

Reply:
@gtrivedi88 Will asciidoc attributes work in this .md file?
I'll double-check that all instances of App Studio are replaced with {ProductName} in .adoc files.